tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1292,6 +1293,38 @@ remove_dead_instrs(void)
   }
 }
 
+/**
+   \brief process debug info of variables with parameter attribute.
+ */
+void
+process_params(void)
+{
+  unsigned smax = stb.stg_avail;
+  for (SPTR sptr = get_symbol_start(); sptr < smax; ++sptr) {
+    if (STYPEG(sptr) == ST_PARAM && should_preserve_param(sptr)) {
+      DTYPE dtype = DTYPEG(sptr);
+      if (DTY(dtype) == TY_ARRAY || DTY(dtype) == TY_STRUCT) {
+        /* array and derived types have 'var$ac' constant variable
+         * lets use that, by renaming that to 'var'.
+         */
+	SPTR new_sptr = (SPTR) CONVAL1G(sptr);
+	NMPTRP(new_sptr, NMPTRG(sptr));
+      } else {
+	LL_DebugInfo *di = cpu_llvm_module->debug_info;
+	int fin = BIH_FINDEX(gbl.entbih);
+	LL_Type *type = make_lltype_from_dtype(dtype);
+	OPERAND *ld = make_operand();
+	ld->ot_type = OT_MDNODE;
+	ld->val.sptr = sptr;
+	LL_MDRef lcl = lldbg_emit_local_variable(di, sptr, fin, true);
+
+        /* lets generate llvm.dbg.value intrinsic for it.*/
+	insertLLVMDbgValue(ld, lcl, sptr, type);
```

Comment:
Alignment.

Reply:
Thanks for your comment. Incorporating it.